_docs/development.md

```diff
@@ -58,8 +61,15 @@ Now clone the RapidPro repository and link up the development settings:
 
 {% highlight bash %}
 $ git clone git@github.com:rapidpro/rapidpro.git
-$ cd rapidpro
-$ ln -s temba/settings.py.dev temba/settings.py
+$ cd rapidpro/temba
+$ ln -s settings.py.dev settings.py
+{% endhighlight %}
+
+At this point, you should edit the rapidpro/settings.py to adjust the password for the postgresql connection.
+
+##Install Node 
+{% highlight bash %}
+$ sudo apt-get install node node-less coffee-script
```

Comment:
Again, this belongs in the prerequisities I think. Maybe we can just add something about "you should be able to type lessc and have it work".

We don't want to get into maintaining instructions on how to install the latest version of all our dependencies, rather link off to those projects.